week1/vue_1/src/App.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <h3>1주차 Vue 실습 - 할 일</h3>
+  <AddingInput @addToDo="addToDo"/>
+  <TodoCard v-for="todo,i in todoList" :key="i" v-bind:todo="todo" @editToDo="editToDo" @toggleToDo="toggleToDo" @deleteToDo="deleteToDo"/>
+
+</template>
+
+<script>
+import AddingInput from './components/AddingInput.vue';
+import TodoCard from './components/TodoCard.vue';
+
+export default {
+  name: 'App',
+
+  components: {
+    AddingInput, TodoCard
+  },
+
+  data() {
+    return {
+      idCnt: 0,
+      todoList: []
+    };
+  },
+  
+  methods: {
+    addToDo(title) {
+      this.todoList.push({
+          id: this.idCnt,
+          title: title,
+          done: false
+      });
+      this.idCnt += 1;
```

Comment:
```suggestion
      this.todoList.push({
          id: createId(),
          title: title,
          done: false
      });
```
아예 id를 난수로 생성해서 넣는 방법도 있답니다 ㅋㅋ
createId를 통해서 id 생성 로직을 추상화하여 관리하는 것도 좋은 방법일 것 같아요!